subprojects/enterprise-operations/src/main/java/org/gradle/internal/taskgraph/NodeIdentity.java

```diff
@@ -20,12 +20,14 @@
  * The identity of a node in the execution graph.
  * <p>
  * Needs to be unique across all the execution graphs of the current build.
+ *
+ * @since 8.1
  */
 public interface NodeIdentity {
 
     enum NodeType {
         TASK,
-        ARTIFACT_TRANSFORM
+        TRANSFORM_STEP
```

Comment:
```suggestion
        PLANNED_TRANSFORM_STEP
```

🤔 Should this be `PLANNED_TRANSFORM_STEP` or is the `Planned` implied since this is part of the task graph. I suppose similarly to task we can drop the `Planned` in the enum like you did. Any thoughts?

Reply:
Yes, I think `Planned` here is implied by the `NodeType` enum, so I would keep it as just `TRANSFORM_STEP`.

> I suppose similarly to task we can drop the Planned in the enum like you did

Not sure, I understand from which enum you suggest dropping this?

Reply:
I don't suggest dropping anything more, I suggest keeping is at it is here.